api/app/controllers/mno_enterprise/jpi/v1/notifications_controller.rb

```diff
@@ -0,0 +1,39 @@
+module MnoEnterprise
+  class Jpi::V1::NotificationsController < Jpi::V1::BaseResourceController
```

Comment:
Authorisation? :policeman:

Reply:
What do we need to add ? All roles can access to their notifications ?